src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200103101609_Routine_UpsertFilter.sql

```diff
@@ -0,0 +1,17 @@
+CREATE PROCEDURE UpsertFilter @Filter dbo.FilterType READONLY
```

Comment:
I'm going to change all procedures and types to `CREATE OR UPDATE`. In the Developer exports (0021 and 0022) we've already got four procedures and their corresponding types presumably from early work on this feature and the Data API  isn't going to start up without either re-issuing a new version to everyone without them included, or this change to `CREATE OR UPDATE`.